renderdoc/driver/vulkan/wrappers/vk_device_funcs.cpp

```diff
@@ -2538,6 +2547,15 @@ bool WrappedVulkan::Serialise_vkCreateDevice(SerialiserType &ser, VkPhysicalDevi
       }
     }
 
+    if (perfQuery)
+    {
+        VkPhysicalDeviceFeatures2 availBase = {VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_FEATURES_2};
+        m_PhysicalDeviceData.performance_query_features.sType = VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_PERFORMANCE_QUERY_FEATURES_KHR;
+        m_PhysicalDeviceData.performance_query_features.pNext = NULL;
+        availBase.pNext = &m_PhysicalDeviceData.performance_query_features;
+        ObjDisp(physicalDevice)->GetPhysicalDeviceFeatures2(Unwrap(physicalDevice), &availBase);
+    }
```

Comment:
This will need to tweak slightly - there's no requirement for a driver supporting the extension to also support the physical device features so we need to check that the feature is available before enabling it. We also don't need `performanceCounterMultipleQueryPools` I don't think, so we shouldn't enable it unless we do.

If you look at how VK_EXT_transform_feedback is handled that's an example of an extension which is conditionally enabled depending on physical device feature.

Reply:
Done.